Directory.Build.props

```diff
@@ -44,9 +44,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- ProjectDir needs to be set for build-tools and legacy usages. New usages should use RepoRoot -->
-    <ProjectDir>$(RepoRoot)</ProjectDir>
-    <SourceDir>$(RepoRoot)src/</SourceDir>
+    <SetupRoot>$(RepoRoot)</SetupRoot>
+    <!-- ProjectDir needs to be set for build-tools and legacy usages. New usages should use SetupRoot. -->
+    <ProjectDir>$(SetupRoot)</ProjectDir>
```

Comment:
I think we can remove this property now, right?

Reply:
There might still be some usages of it that need to be updated. I’ll go do a pass next week and remove them.